### ISSUE TITLE
no-hashtag-description: Allow comments between described fields

### ISSUE THREAD

**Issue by tobiasdiez** [dotansimha/graphql-eslint#570]
**Describe the bug**

Comments between fields with a description are still marked as an error, e.g. in
```graphql
interface Document {
  "The ID of the document."
  id: ID!

  # This comment triggers an error message

  "Some other field."
  field: [String]
}
```

**To Reproduce**
Run the plugin with the no-hashtag-description rule enabled on the above graphql schema.

**Expected behavior**

No error message, since the purpose of this rule (if I'm not mistaken) is to enforce the usage of ticks for the documentation. But this should still allow hashtags for comments, which wouldn't show up for example in a documentation.

**Environment:**

- OS: 
- `@graphql-eslint/...`: 1.1.4
- NodeJS:

**Additional context**

**Comment by B2o5T** [comment]
Thanks, you can try `2.0.2-alpha-6b1ad82.0` alpha version

**Comment by tobiasdiez** [comment]
Thanks for the quick fix! I tried the new version and can confirm that the original error disappeared.
However, now the above code results in the error `Expected space or tab before '*/' in comment. eslint spaced-comment` at the comment. Moreover, double sharps as in `## Test` trigger `Expected space or tab before '*/' in comment.` in addition. I'm not sure if that's an issue with the plugin, or if one should disable eslint's spaced-comment rule for graphql.

**Comment by B2o5T** [comment]
I think you need to disable `spaced-comment` rule for `graphql` files or move javascript rules to `overrides` section for `js` files as `spaced-comment` lint javascript comments, and not graphql comments

**Comment by tobiasdiez** [comment]
I'm not sure which plugin is configuring this rule (is there an easy way to find this out?). The following is my config:
```
  extends: [
    '@nuxtjs/eslint-config-typescript',
    'plugin:@typescript-eslint/recommended',
    'prettier',
    'plugin:prettier/recommended',
    'plugin:nuxt/recommended',
  ],
```
https://github.com/JabRef/JabRefOnline/blob/main/.eslintrc.js

**Comment by B2o5T** [comment]
ESLint treat `## Test` comment like `# Test` string, use single hash for comments if you want to use `spaced-comment` rule on graphql files.

I don't get any warnings on this code with `spaced-comment` rule enabled
```gql
interface Document {
  "The ID of the document."
  id: ID!

  # This comment triggers an error message

  "Some other field."
  field: [String]
}
```

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@2.1.0`. Thanks @B2o5T !
